lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -2260,7 +2269,7 @@ bool DWARFASTParserClang::CompleteTypeFromDWARF(const DWARFDIE &die,
 
   // Disable external storage for this type so we don't get anymore
   // clang::ExternalASTSource queries for this type.
-  m_ast.SetHasExternalStorage(clang_type.GetOpaqueQualType(), false);
+  //m_ast.SetHasExternalStorage(clang_type.GetOpaqueQualType(), false);
```

Comment:
This should not be in here and I will revert this. I was testing to see if I removed this, if I would get a `find 'B' in 'A'` call to the `ClangASTSource::FindExternalVisibleDecls(...)`, but it never happened. I was hoping we could just leave the external AST source enabled for classes and that would allow us to find the contained types efficiently, so that was my original approach.